lstchain/datachecks/containers.py

```diff
@@ -35,6 +40,8 @@ class DL1DataCheckContainer(Container):
     num_ucts_jumps = Field(-1, 'Number of observed (and corrected) UCTS jumps')
     mean_alt_tel = Field(None, 'Mean telescope altitude')
     mean_az_tel = Field(None, 'Mean telescope azimuth')
+    tel_ra = Field(None, 'Telescope pointing RA (deg)')
+    tel_dec = Field(None, 'Telescope pointing declination')
```

Comment:
container fields should have the correct defaults, so that if a default container is written, the correct tables are setup. None will result in the column being skipped. I think these (and the above) should be `np.nan * u.deg` and `ucts_trigger_type` and `trigger_type` should probably be `0` (they are bitfields of which at least one bit must be set to be a valid trigger type, so 0 would make sense as invalid value marker.

Reply:
I comment here because I cannot comment on the old code. But to setup a logger related to `ctapipe.instrument.camera` in a container class definition is really strange and should probably be (re)moved.

Reply:
Having this in the class body get's executed at import time. So this will set it for the whole of lstchain, as soon as `lstchain/datachecks/container` is imported. If you need this for some reason, I guess connected to https://github.com/cta-observatory/ctapipe/issues/1828, you should set this in the main method of the script, not here.

Reply:
> I comment here because I cannot comment on the old code. But to setup a logger related to `ctapipe.instrument.camera` in a container class definition is really strange and should probably be (re)moved.

I don't remember why I put that there in the first place... gone!

Reply:
> 

Sorry, I don't see what does this comment refer to, can you clarify?

Reply:
> Having this in the class body get's executed at import time. So this will set it for the whole of lstchain, as soon as `lstchain/datachecks/container` is imported. If you need this for some reason, I guess connected to [cta-observatory/ctapipe#1828](https://github.com/cta-observatory/ctapipe/issues/1828), you should set this in the main method of the script, not here.

Sorry, I don't see what does this comment refer to, can you clarify?

Reply:
This logging setup